packages/web-frontend/cypress/scripts/generateConfig/generateReportConfig.js

```diff
@@ -3,212 +3,62 @@
+  const path = [project, orgUnit, dashboardGroup].map(encodeURIComponent).join('/');
+  const queryParams = { report: id, reportPeriod: buildReportPeriod(startDate, endDate) };
 
-    if (orgUnitCode && projectCode) {
-      const reportPeriod = selectPeriod(viewJson);
-      return { dashboardGroup, orgUnitCode, projectCode, reportPeriod };
-    }
-  }
-
-  // No compatible entry found, throw error
-  throw new Error(
-    [
-      `No compatible org unit map entry found for report '${report.id}'`,
-      'Try using one of the following entries:',
-      ...attemptedMapEntries.map(({ key, level }) => `* Key: '${key}', level: '${level}'`),
-    ].join('\n'),
-  );
-};
-
-const getUrlsForReports = async (db, reports, reportIdToGroups) => {
-  const skippedReports = Object.fromEntries(
-    Object.values(WARNING_TYPES).map(warnType => [warnType, []]),
-  );
-  const addSkippedReport = (warnType, reportDescription) => {
-    skippedReports[warnType].push(reportDescription);
-  };
-
-  const getUrlForReport = async report => {
-    const { dataBuilder, drillDownLevel } = report;
-
-    if (drillDownLevel) {
-      addSkippedReport(WARNING_TYPES.DRILL_DOWN, `${report.id} - level ${drillDownLevel}`);
-      return null;
-    }
-
-    if (!dataBuilder) {
-      addSkippedReport(WARNING_TYPES.NO_DATA_BUILDER, report.id);
-      return null;
-    }
-
-    const groupsForReport = reportIdToGroups[report.id];
-    if (!groupsForReport) {
-      addSkippedReport(WARNING_TYPES.NO_GROUP, report.id);
-      return null;
-    }
-
-    if (!groupsForReport.some(dg => dg.projectCodes)) {
-      addSkippedReport(WARNING_TYPES.NO_PROJECT, report.id);
-      return null;
-    }
-
-    const urlParams = await selectUrlParams(db, report, groupsForReport);
-    return createUrl(report, urlParams);
-  };
-  const urls = await Promise.all(reports.map(getUrlForReport));
-
-  return {
-    urls: urls.filter(u => u).sort(compareAsc),
-    skippedReports: filterValues(skippedReports, r => r.length > 0),
-  };
-};
-
-const getReportIdToGroups = async db => {
-  const dashboardGroups = await db.executeSql(`SELECT * from "dashboardGroup"`);
-
-  const reportIdToGroups = {};
-  dashboardGroups.forEach(dashboardGroup => {
-    dashboardGroup.dashboardReports.forEach(reportId => {
-      if (!reportIdToGroups[reportId]) {
-        reportIdToGroups[reportId] = [];
-      }
-      reportIdToGroups[reportId].push(dashboardGroup);
-    });
-  });
-  return reportIdToGroups;
+  return stringifyQuery('', path, queryParams);
 };
 
-/**
- * We generate the least amount of config required to test each report once.
- * 1. For each report, use the last dashboard group that includes it
- * 2. For each dashboard group, use the first project that includes it
- * 3. For each dashboard group, use an organisation unit from the orgUnitMap config
- * that matches the group's org unit code and level
- */
-export const generateReportConfig = async db => {
+export const generateReportConfig = () => {
```

Comment:
Two notes for this function:

1. I believe in the future we may still want to use the dynamic url building functionality that is deleted here. A use case is: if a specific project is selected
  * find all reports for that project
  * for each report, try crawlling through org units and periods until data is found (in the front end). We could configure a period/org unit range per project, striking a balance between ease of configuration and performance
  
    But this is just food for thought - for now, lets keep it simple and delete whatever is not required

2. Currently, the user must first run `yarn cypress:config` to build the report urls, then run the tests. You may ask: why not build the urls inside the tests themselves? There are two reasons for that:
  i. We may want to enable more elaborate filtering/url building. Eg I adding a `--project strive` option in our e2e test runner could be helpful
  ii. The test runner should have an overview of the url count before tests run, if we are to introduce parallelization

Reply:
1. Totally agree, and we can always find this code in the version history
2. Makes sense. I would like to keep an eye on this one, and see if it strikes the right balance in practice. An alternative could be something like building as a postinstall step, so that we're actually committing the generated config at least as the default urls, or adding a `yarn cypress:build-and-run` script. But best to get this into the wild and see how people actually use it and whether there are pain points